README.md

```diff
@@ -1,9 +1,12 @@
 # DPNS Contract
 
-[![Build Status](https://img.shields.io/travis/com/dashevo/dpns-contract.svg?branch=master&style=flat-square)](https://travis-ci.com/dashevo/dpns-contract)
-[![NPM version](https://img.shields.io/npm/v/@dashevo/dpns-contract.svg?style=flat-square)](https://npmjs.org/package/@dashevo/dpns-contract)
+[![NPM Version](https://img.shields.io/npm/v/@dashevo/dpns-contract)](https://www.npmjs.com/package/@dashevo/dpns-contract)
+[![Build Status](https://travis-ci.com/dashevo/dpns-contract.svg?branch=master)](https://travis-ci.com/dashevo/dpns-contract)
+[![Release Date](https://img.shields.io/github/release-date/dashevo/dpns-contract)](https://github.com/dashevo/dpns-contract/releases/latest)
```

Comment:
Currently there are no actual releases created on Github even though there is one on npm. Probably should make the tag matching the npm version into an actual Github release so this badge works.